### DISCUSSION TITLE
Welcome to bigspicy Discussions!

### DISCUSSION THREAD

**Discussion by growly** [google/bigspicy#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Abide by our [code of conduct](https://github.com/google/bigspicy/blob/34c2b1ff8a7d2a7d33514fd5875643d283c40b94/docs/code-of-conduct.md).

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AndyNotfound** [comment]
Hi, you can call me Andy. I'm in my sophomore year at Batam Institute of Technology. I've been wondering for a while about this. Let's say I applied for a remote job in USA, how will they pay my salary? Will they transfer it to my local bank?

**Comment by paxdev55** [comment]
I like the idea of these discussions. I have a question- is the "big spicy" repository supposed to be the Google hub on GH?

I'm Dino and I still go to school. I'm pansexual, too.

**Comment by growly** [comment]
This forum is for technical discussion of bigspicy, a tool for merging circuit descriptions (netlists), generating Spice decks modeling those circuits, generating Spice tests to measure those models, and analyzing the results of running Spice on those tests. It is not for general discussion about Google.
